lessons/python/intro/IntroPython-AH.ipynb

```diff
@@ -108,6 +116,13 @@
     "name = 'Ahmed' # this is a string"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can then use _functions_ on our variables to do things with them. The `print` function returns the value of a variable:"
```

Comment:
`print` returns `None`; might have to differentiate between output and what the cell prints by default.
```suggestion
    "We can then use _functions_ on our variables to do things with them. The `print` function outputs the value of a variable:"
```

Reply:
Very true, important distinction - on it